From: Gerald Schaefer <geraldsc@de.ibm.com>
Subject: kmsg: do not change pr_xyz messages without KMSG_COMPONENT
References: bnc#445100

Symptom:     Normal pr_xyz messages without KMSG_COMPONENT have an additional
             ": " in the output.
Problem:     The pr_printk macro if KMSG_COMPONENT is not defined has 
             additional text.
Solution:    Remove the additional text.

Acked-by: John Jolly <jjolly@suse.de>

---
 include/linux/kernel.h |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/include/linux/kernel.h
+++ b/include/linux/kernel.h
@@ -301,7 +301,7 @@ static inline char *pack_hex_byte(char *
 	printk(level KMSG_COMPONENT  ": " format, ##__VA_ARGS__)
 #else
 #define pr_printk(level, format, ...) \
-	printk(level ": " format, ##__VA_ARGS__)
+	printk(level format, ##__VA_ARGS__)
 #endif
 
 #if defined(__KMSG_CHECKER) && defined(KMSG_COMPONENT)
